applications.json

```diff
@@ -7111,6 +7111,20 @@
             "languages" : [
                 "rust"
             ]            
+        },
+        {
+            "short_description": "Cross-platform development IDE",
+            "categories": [
+                "IDE"
```

Comment:
Categories name should be lowercased.

Reply:
hmm... need to adjust my github notifications... I'm just seeing this now.